lib/rsync_store.ml

```diff
@@ -144,7 +144,7 @@ let cache ~user t name =
   end >>= fun () ->
   (* Create writeable clone. *)
   let gen = cache.gen in
-  let { Obuilder_spec.uid; gid } = user in
+  let { Obuilder_spec.uid; gid } = match user with `Unix user -> user | _ -> assert false in
```

Comment:
Same suggestion here, please exhaustively pattern match.
rsync could work on Windows it is not implemented yet.